deploy/podman/README.md

```diff
@@ -22,6 +22,14 @@ will connect to.
 podman play kube --configmap configmap.yml pod.yml
 ```
 
+To preserve data about existing clusters between pod reconfigurations the
```

Comment:
Since the podman runtime method is intended for one-time use to deploy a single cluster, I think we should add a little guidance here about when the user would want to use this option for persistence. What use case motivated this PR?

Reply:
The reason behind PR is mainly as stated in commit message, I would like to manipulate the release and os images, for example extend available versions served by AI but preserving information about already deployed clusters. Using podman to host AI works good in my dev environment, it`s quick and does not require too much dependencies. Preserving data after pod recreation is useful for me because I can still refer to existing clusters.

Reply:
Ok great. It's good to hear that it's proving useful.

I think it'll be helpful to provide a little context for new users so they can understand which way to go, and to not just default to "well if I can have persistence for free, that sounds like a good thing, so I guess I'll do that". How about a sentence something like this?

"If you only want to provision a single cluster at a time, which is a common use case, persistence is not necessary."

Reply:
Ok, added.